qiskit/transpiler/passes/utils/filter_op_nodes.py

```diff
@@ -0,0 +1,63 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2023.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Remove all babeled ops from a circuit"""
```

Comment:
lol

Reply:
Fixed in: https://github.com/Qiskit/qiskit/pull/10323/commits/f6df5be6021c7acd9c28151f9278631c7d203e78